src/Actions/Action.php

```diff
@@ -147,16 +148,24 @@ public function handleRequest(ActionRequest $request)
             $request->collectRepositories($this, static::$chunkCount, function ($models) use ($request, &$response) {
                 Transaction::run(function () use ($models, $request, &$response) {
                     $response = $this->handle($request, $models);
+
+                    $models->each(fn (Model $model) => ActionLog::forRepositoryAction($this, $model, $request->user())->save());
```

Comment:
We should have an option to turn this off... also a command to prune the saved logs